staging/src/k8s.io/pod-security-admission/test/fixtures_dropCapabilities.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/pod-security-admission/api"
+)
+
+/*
+TODO: include field paths in reflect-based unit test
+containerFields: []string{
+	`securityContext.capabilities.drop`,
+},
+*/
+
+func init() {
+	fixtureData_1_22 := fixtureGenerator{
+		expectErrorSubstring: "drop all",
+		generatePass: func(p *corev1.Pod) []*corev1.Pod {
```

Comment:
generatePass should return:

- pod with all capabilities dropped, nothing added back
- pod with all capabilities dropped, `CAP_NET_BIND_SERVICE` added back

I'm pretty sure you'll need to update the minimal valid pod as well.

Reply:
yes, minimal valid restricted pod will need to be updated to drop ALL in all containers

Reply:
you'll also need to update fixtures_addCapabilities.go#generatePass

1. inspect the minimal valid pod passed into generatePass
2. IF the minimal valid pod drops `ALL`, just return `[]*corev1.Pod{p}` (or return nil if https://github.com/kubernetes/kubernetes/pull/103517 has already merged), since it means some other check (the one being implemented in this PR) has stricter requirements for capabilities